src/components/Footer/index.jsx

```diff
@@ -1,5 +1,91 @@
+import Link from "next/link";
+import { AiFillFacebook } from "react-icons/ai";
+import { AiFillLinkedin } from "react-icons/ai";
+import { BsShareFill } from "react-icons/bs";
+import { FaInstagram } from "react-icons/fa";
+import { TbLanguage } from "react-icons/tb";
+
 function Footer() {
-  return <div>Footer</div>;
+  return (
+    <div className='bg-neutral py-10 px-6 text-white fixed bottom-0 w-full'>
```

Comment:
## Request for Change Review - "Footer" PR

**Reviewer:** @riadhmouamnia 

Thank you for your work on the "Footer" PR. This is a little late but I've identified a couple of issues that need to be addressed before merging:

1. **Positioning Bug**: The footer currently has a `position: fixed` property in its CSS classes, causing it to overlay other sections and always appear at the bottom of the page.

2. **Branch Update**: The branch is not up to date with the `develop` branch. so we need to rebase or merge the latest changes from `develop` into this branch to resolve any conflicts and ensure alignment with the most recent codebase.

I'll handle the necessary fixes for these issues myself. and then push the changes so don't worry about this issue @rahemSorour and focus on your other task!

@riadhmouamnia